stories/MatComponents/index.js

```diff
@@ -3,10 +3,12 @@ import { storiesOf } from '@storybook/react';
 import { Footer } from '../../src/organisms/Footer';
 import description from './mat-description';
 import matSteps from './mat-steps';
+import { Quantity } from '../../src/atoms/MainRecipe/quantity';
```

Comment:
I think we can avoid including this tiny component to storybook. It's just for reusing in bigger Mat-components and not for reusing it as separate component in wolverine.